src/App.tsx

```diff
@@ -1,61 +1,150 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
 
 export const App = () => {
+  const [todoList, setTodo] = useState(todosFromServer);
+  const [todoState, setData] = useState({
+    title: '',
+    select: '0',
+  });
+  const [titleError, setTitileError] = useState(false);
+  const [selectError, setSelectError] = useState(false);
```

Comment:
```suggestion
  const [isTitleError, setIsTitleError] = useState(false);
  const [isSelectError, setIsSelectError] = useState(false);
```
https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig